src/sentry/sentry_metrics/configuration.py

```diff
@@ -47,4 +55,30 @@ def get_ingest_config(use_case_key: UseCaseKey) -> MetricsIngestConfiguration:
             )
         )
 
-    return _METRICS_INGEST_CONFIG_BY_USE_CASE[use_case_key]
+        # Make sure that if we run backends in parallel we aren't
+        # double producing to the snuba-metrics-generics topic
+        dummy_output_topic = settings.KAFKA_SNUBA_GENERICS_METRICS_DUMMY
+
+        _register_ingest_config(
+            MetricsIngestConfiguration(
+                db_backend=CLOUDSPANNER_DB,
+                input_topic=settings.KAFKA_INGEST_METRICS,
+                output_topic=dummy_output_topic,
+                use_case_id=UseCaseKey.RELEASE_HEALTH,
+                internal_metrics_tag="release-health-spanner",
+                writes_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS,
+            )
+        )
```

Comment:
We do not plan to run a cloudspanner indexer for release health any time soon.  Up to you if you want to have it here or not. No strong opinion.